types/n3/index.d.ts

```diff
@@ -149,8 +150,35 @@ export class Lexer {
     tokenize(input: string): Token[];
     tokenize(input: string | EventEmitter, callback: TokenCallback): void;
 }
+
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types
+export type MimeType =
+  // Discrete types
+  | 'application' | 'example' | 'text'
+  // Multipart types
+  | 'message' | 'multipart';
+
+export type BaseFormat =
+  | 'turtle'
+  | 'triG'
+  | 'N-Triples'
+  | 'N-Quads'
+  | 'n3'
+  | 'Notation3';
```

Comment:
Shall we perhaps keep casing consistent here? Should be `TriG` and `Turtle` and `N3` for me.